### PR TITLE
Update testimonials for KWoC 2022

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
* Removed some old testimonials
* Removed testimonials with broken links
* Added 2020 and 2021 testimonials
* Rearranged some of the testimonials

Credits: 
- @ashwinpra @somya-kr for finding testimonials

### CODE REVIEW

File: src/data/testimonials.json [hunk 1]
```diff
@@ -8,25 +29,53 @@
...
+      "quote": "Participating in KWOC has given me exposure to the open source community as well as a number of other open source programmes. I’ve also picked up a few more GitHub features and Git commands. It’s also given me the confidence to learn new tech tools/frameworks in a limited period of time that are relevant to the projects I’m interested in working on.",
+      "imageLink": "https://avatars.githubusercontent.com/JessicaJohnBritto",
+      "blogLink": "https://medium.com/@jessjohnbritto/kwoc-kharagpur-winter-of-code-project-report-4b7d852ec1e4"
     },
     {
-      "name": "Athitya Kumar",
-      "role": "KWoC '16 and '17' Mentor",
-      "quote": "The experience has been great both the years, with a lot of new students learning open-source development from KWoC by contributing to the projects. Some of my mentees have even gone ahead to do a GSoC project as well in the following summers, and are definitely are still actively contributing to open-source.",
-      "imageLink": "https://avatars.githubusercontent.com/athityakumar",
-      "blogLink": "https://athityakumar.github.io/blog/posts/Kharagpur_Winter_of_Code_2016/"
```
Comment (sahil-shubham) [inline src/data/testimonials.json:29]: Why remove old testimonials when the links are working? I find the old ones valuable as well.

File: src/data/testimonials.json [hunk 2]
```diff
@@ -77,13 +119,6 @@
       "imageLink": "https://avatars.githubusercontent.com/themousepotato",
       "blogLink": "https://medium.com/@themousepotato/kharagpur-winter-of-code-2017-6e350c2252c7"
     },
-    {
-      "name": "Sayan Sinha",
-      "role": "KWoC '16 Student",
-      "quote": "KWoC was a lot more than experience and learning. Now, I had worked on the repositories and had interacted with their makers.",
-      "imageLink": "https://avatars2.githubusercontent.com/u/23132358?s=460&v=4",
-      "blogLink": "https://blog.kossiitkgp.org/kharagpur-winter-of-code-c4eeb4af71f1"
```
Comment (sahil-shubham) [inline src/data/testimonials.json:85]: This link is working instead - https://medium.com/kharagpur-open-source-society/kharagpur-winter-of-code-c4eeb4af71f1

Sayan, Yash and Shivam are our advisors, while Athitya was a valuable part of MetaKGP. I think we can keep their feedback if the links are working.

**Review by sahil-shubham — APPROVED** [review]

File: src/data/testimonials.json (hunk 2 above)
Comment (harshkhandeparkar) [reply src/data/testimonials.json:85]: Will update Sayan's link and add the Advisors' testimonials. 
Shivam's link is broken too, is there a working link for it?

**Comment by sahil-shubham** [comment]
Looks good, thanks for adding back the old testimonials. Merging.

**Pull request merged**